java/src/test/java/ai/rapids/cudf/CuFileTest.java

```diff
@@ -102,4 +102,55 @@ private void verifyAppendToFile(File tempFile) {
       assertEquals(987654321, dest.getLong(0));
     }
   }
+
+  @Test
+  public void testRegisteringUnalignedBufferThrowsException() {
+    assumeTrue(CuFile.libraryLoaded());
+    assertThrows(IllegalArgumentException.class, () -> {
+      try (DeviceMemoryBuffer buffer = DeviceMemoryBuffer.allocate(4095)) {
+        new CuFileBuffer(buffer, true);
```

Comment:
Should use try-with-resources in case it does _not_ throw.
```suggestion
      try (DeviceMemoryBuffer buffer = DeviceMemoryBuffer.allocate(4095);
           CuFileBuffer ignored = new CuFileBuffer(buffer, true)) {
```

Reply:
Done.